src/components/specimen/components/idCard/IDCard.tsx

```diff
@@ -0,0 +1,131 @@
+/* Import Dependencies */
+import { Row, Col, Card } from 'react-bootstrap';
+
+/* Import Store */
+import { useAppSelector } from 'app/hooks';
+import { getSpecimen } from "redux/specimen/SpecimenSlice";
+
+/* Import Icons */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faFrog } from '@fortawesome/free-solid-svg-icons'
+
+/* Import Components */
+import MidsBar from './MidsBar';
+
+
+const IDCard = () => {
+    /* Base variables */
+    const specimen = useAppSelector(getSpecimen);
+
+    const logo = (ror: string) => {
+        try {
+            return require(`../../../../webroot/img/organizationLogo/${ror}.png`);
```

Comment:
Does this mean we have to have all logo's in this folder?

Reply:
https://naturalis.atlassian.net/browse/DD-265?atlOrigin=eyJpIjoiYjA4NDg4NTcwYzMzNDRkN2I3ZWRlNmQ1MWFiYTY3MjMiLCJwIjoiaiJ9